x/auth/tx/msg_service.go

```diff
@@ -19,6 +19,11 @@ func (w *wrapper) Invoke(_ gocontext.Context, method string, args, reply interfa
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidType, "%T should implement %T", args, (*sdk.MsgRequest)(nil))
 	}
 
+	err := req.ValidateBasic()
+	if err != nil {
+		return err
```

Comment:
shouldn't we use `sdkerrors.Wrapf` here?

Reply:
I thumbed up, but maybe it's not necessary. err should already be a SDK error. I used `sdkerrors.Wrapf` because there's no underlying error.